mksnapshot.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/sh -x
```

Comment:
Your answer does not compute for me because when `-x` is needed (and I am not disputing or even discussing its need) it should go into the _caller_, not the script itself.

This is not a blocking issue for me.